### ISSUE TITLE
Can not trace exactly QRcode Image

### ISSUE THREAD

**Issue by huynhnhathoangit** [jankovicsandras/imagetracerjava#11]
Can not trace exactly QRcode Image, there is an error inside QRCODE image when trace it.
![image](https://user-images.githubusercontent.com/28619787/174017121-ab4d9994-3d1a-4863-9fbd-67f502ab3dc8.png)

**Comment by jankovicsandras** [comment]
Hi,

I tested this command below and it gave me an acceptable SVG of the QR code. Can you please explain the problem with more details?

```
java -jar ImageTracer.jar qr.png
```

![qr png](https://user-images.githubusercontent.com/12699003/174298693-8f32e3a5-2d53-4a41-84c5-a92d562ae969.svg)

**Comment by huynhnhathoangit** [comment]
Hello @[jankovicsandras]
Thanks for your reply.
I problem is the output SVG file of QR code was broken and can not SCAN,
As you can see, the left side is origin QR code can Scan with camera: value: 0, but the right side have error some part inside, so can not scan.
![image](https://user-images.githubusercontent.com/28619787/174511778-7d893a98-7500-4636-8f46-143b13aab9dd.png)

**Comment by jankovicsandras** [comment]
Yes, I see your problem: some white square "holes" are filled black on the right side, which destroys the QR code. I guess the problem is with the colors: the source might have transparency instead of white, and despite ImageTracer probably _can_ trace these "hole" squares correctly, this older Java version is using stack based rendering, which means transparent on top of black is still black.

![Stack transparency problem](https://raw.githubusercontent.com/jankovicsandras/imagetracerjs/master/docimages/transparency_problem.png)

See also [imagetracerjs Issue Seven](https://github.com/jankovicsandras/imagetracerjs/issues/7)
This is fixed in imagetracerjs 1.2.0 , but this Java version is 1.1.2 . I'm planning to port this fixes to the Java versions, but I don't know when I will have time. 

Despite all of this, I _still_ got acceptable result by saving your left side PNG, and tracing with the command above, so you might be able to fix this also with another color palette.

 
![qr2 png](https://user-images.githubusercontent.com/12699003/177865118-4c227d8c-191c-4427-ac14-ac4b7df1f392.svg)


Can you please upload this right side image as the SVG result from ImageTracer (so do not convert it to PNG)?
